crates/story/src/sidebar_story.rs

```diff
@@ -310,23 +311,35 @@ impl Render for SidebarStory {
                     .child(
                         SidebarGroup::new("Platform").child(SidebarMenu::new().children({
                             let mut items = Vec::with_capacity(groups[0].len());
-                            for item in groups[0].iter() {
+                            for (ix, item) in groups[0].iter().enumerate() {
                                 let item = *item;
+                                let id =
+                                    ElementId::Name(SharedString::new(format!("item-{}", ix,)));
+
                                 items.push(
-                                    SidebarMenuItem::new(item.label(), item.label())
+                                    SidebarMenuItem::new(item.label())
+                                        .id(id)
                                         .icon(item.icon().into())
                                         .active(self.active_item == item)
                                         .children({
                                             let mut sub_items =
                                                 Vec::with_capacity(item.items().len());
-                                            for sub_item in item.items() {
+                                            for (ix, sub_item) in
+                                                item.items().into_iter().enumerate()
+                                            {
+                                                let id = ElementId::Name(SharedString::new(
+                                                    format!("sub-item-{}", ix),
+                                                ));
```

Comment:
You're not need to format ix as a string.

GPUI internal will use all parent ids to build a nested id for global_id. So here we can just use `ix` as the item id.

Reply:
I am going to update this directly, you can see later what I am changed.

Reply:
as you wish 😄 . No clue why it did not work for me, I guess I made a typo somewhere. Thanks!

Reply:
Hmm for me still not working properly. I'm curious on how you fix it 🙇